core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -584,7 +584,7 @@ func (ec *EthConfirmer) batchFetchReceipts(ctx context.Context, attempts []EthTx
 		// Counters are prune to being in-accurate due to re-orgs.
 		if ec.config.EvmUseForwarders() {
 			meta, err := attempt.EthTx.GetMeta()
-			if err != nil {
+			if err != nil || meta == nil {
```

Comment:
Needs a regression test

Reply:
Addressed both comments. Regression test added and moved the PR descriptiion into commit message body